public/API/API_GetLeaderboardSubmissions.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ *  API_GetLeaderboardSubmissions - returns a info and submissions for a Leaderboard for the given ID
+ *    i : leaderboardID
+ *    o : offset - number of entries to skip (default: 0)
+ *    c : count - number of entries to return (default: 100, max: 500)
+ *  string      Title                       name of the leaderboard
+ *  int         Description                 details about what the leaderboard is tracking
+ *  int         Count                       number of user entries returned in the response
+ *  int         Total                       number of user entries the leaderboard actually has overall
+ *  array       Entries
+ *   object      [value]
+ *    int        Rank                       user's leaderboard rank
+ *    string     User                       name of user
+ *    string     Score                      string value of the proper ValueFormat of the leaderboard entry //this feels wrong
+ *    string     DateSubmitted              an ISO8601 timestamp string for when the entry was submitted
```

Comment:
Recommend nesting entries metadata into a unique object like:

```ts
{
  "Entries": {
    "Count": 100,
    "Total": 512,
    "Results": [ ... ]
  }
}

```

Reply:
Done, but not sure that `$entries = new stdClass();` is the right way?